src/stactools/sentinel2/stac.py

```diff
@@ -174,8 +175,24 @@ def create_item(
         raise ValueError(
             f"Could not determine EPSG code for {granule_href}; which is required."
         )
-    centroid = shapely.geometry.shape(item.geometry).centroid
-    projection.centroid = {"lat": round(centroid.y, 5), "lon": round(centroid.x, 5)}
+
+    if item.geometry and item.geometry.get("type") == "MultiPolygon":
+        normalized_centroid_geometry = json.loads(json.dumps(item.geometry.copy()))
+        for c1 in normalized_centroid_geometry["coordinates"]:
+            for c2 in c1:
+                for c3 in c2:
+                    if c3[0] < -160:
```

Comment:
```suggestion
                    if c3[0] < -180:
```
I think you mean -180 here, right?

Reply:
no, because it will never be less than -180. The idea was only to normalize the points within the multipolygon that were close the antimeridian from the negative lon side. 

Though thinking about this more, it should probably just be < 0, since the antimeridian-crossing case is the only one that produces a MultiPolygon (I think?).

Reply:
>Though thinking about this more, it should probably just be < 0, since the antimeridian-crossing case is the only one that produces a MultiPolygon (I think?).

If that is correct, then < 0 is the way to go.